test/cases/compile_errors/address_of_threadlocal_not_comptime_known.zig

```diff
@@ -10,4 +10,5 @@ pub export fn entry() void {
 // target=native
 //
 // :2:36: error: unable to resolve comptime value
-// :2:36: note: container level variable initializers must be comptime-known
+// :2:36: note: global variable initializer must be comptime-known
+// :2:36: note: thread local and dll imported variables have runtime-known addresses
```

Comment:
should this say "threadlocal" ?
also what is "dll imported" mean in this context?

Reply:
This error message already existed, it just only appeared in certain scenarios; I just unified it with another, equivalent, error. Changing its wording is outside the scope of this PR.

"DLL imported" relates to the `is_dll_import` field of `std.builtin.ExternOptions`.